### PR TITLE
Create mattermost-editorial-calendar.md

### PR DESCRIPTION
**Pull request by aedott**
#### Summary
Adding a section on how to use our updated editorial calendar to the Handbook!

### CODE REVIEW

**Review by jasonblais — DISMISSED** [review]
This is excellent, thank you! Will defer editor review to Justine.

@justinegeffen can you also work with Ashley to add this into the LHS navigation?

**Comment by aedott** [comment]
Thanks @jasonblais! @justinegeffen should I create an additional pull request to add this page to the navigation?

**Comment by jasonblais** [comment]
@aedott At the bottom of this page, if you click on the branch you created (`editorial-calendar`), this will open GitHub in that branch.


From there, any changes you make will be directly applied to this same pull request.

Let me know if I can help clarify :)

**Comment by aedott** [comment]
Okay! Hoping I did that correctly. @justinegeffen  I'm not sure how to resolve the conflicts -- sorry for being a Github n00b :)

**Comment by justinegeffen** [comment]
No problem at all, @aedott! :D

File: SUMMARY.md [hunk 1]
```diff
@@ -113,6 +113,7 @@
   * [Demand Generation Reporting](operations/messaging-and-math/demand-generation-reporting.md)
   * [M&M Asana Guidelines](operations/messaging-and-math/asana-guidelines.md)
   * [Content Marketing](operations/messaging-and-math/content-marketing.md)
+    * [How to use our editorial calendar](operations/messaging-and-math/content-marketing/editorial-calendar.md)
```
Comment (justinegeffen) [inline SUMMARY.md]:
```suggestion
    * [How to Use the Editorial Calendar](operations/messaging-and-math/content-marketing/editorial-calendar.md)
```

File: operations/messaging-and-math/content-marketing/mattermost-editorial-calendar.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
+# How to Use the Mattermost Editorial Calendar
+
+Our editorial calendar can be found in [Asana](https://app.asana.com/0/1190898966199137/board), and is used to track upcoming content to be published on the Mattermost website, blog, developer blog, and third party channels. 
+
+## Templates
+The editorial calendar includes a Templates section, with workflow templates for common content types. To create a new task using these templates, right click the Template you want to use and select "Duplicate Task." The task will automatically be added to "Inbound Topics."
```
Comment (justinegeffen) [inline operations/messaging-and-math/content-marketing/mattermost-editorial-calendar.md]:
```suggestion
The editorial calendar includes a Templates section, with workflow templates for common content types. To create a new task using these templates, right click the Template you want to use and select **Duplicate Task**. The task will automatically be added to **Inbound Topics**.
```

File: operations/messaging-and-math/content-marketing/mattermost-editorial-calendar.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+
+## Task Workflow
+
+Every piece of content moves through the same general workflow for creation, review, scheduling and distribution, which is reflected in the "Board View" of the Asana project.
```
Comment (justinegeffen) [inline operations/messaging-and-math/content-marketing/mattermost-editorial-calendar.md]:
```suggestion
Every piece of content moves through the same general workflow for creation, review, scheduling, and distribution, which is reflected in the **Board View** of the Asana project. 
```

**Review by jasonblais — DISMISSED** [review]
:+1:

**Review by jasonblais — APPROVED** [review]

**Review by justinegeffen — APPROVED** [review]

**Pull request merged**